chromadb/types.py

```diff
@@ -35,6 +68,97 @@ class Collection(TypedDict):
     # in single-node chroma, this field is always 0
     version: int
 
+    def __init__(
+        self,
+        id: UUID,
+        name: str,
+        configuration: CollectionConfiguration,
+        metadata: Optional[Metadata],
+        dimension: Optional[int],
+        tenant: str,
+        database: str,
+        version: int = 0,
+    ):
+        super().__init__(
+            id=id,
+            name=name,
+            metadata=metadata,
+            configuration_json=configuration.to_json(),
+            dimension=dimension,
+            tenant=tenant,
+            database=database,
+            version=version,
+        )
+
+    def modify(
```

Comment:
Is this modify method used anywhere?

Reply:
I don't think so, looks like it was implemented and never used.